x/feegrant/go.mod

```diff
@@ -158,3 +158,4 @@ require (
 // Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
 // TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
 replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+replace github.com/cosmos/cosmos-sdk => ../../
```

Comment:
We can remove this once we tag

Reply:
We won't take the SDK in a while, best to use a pseudo version from this branch.

Reply:
Yeah, I realized later that we are not tagging x/auth separately